### ISSUE TITLE
Custom Fields in Relationship Processor

### ISSUE THREAD

**Issue by danaskallman** [mecachisenros/caldera-forms-civicrm#12]
Wondering if having custom fields in the Relationship Processor can be done? 

One use case is adding an Emergency Contact Relationship and asking what that relationship is, so a text box (but could be anything really).

**Comment by mecachisenros** [comment]
@danaskallman yes, that can definitely be done, I have not thought  about it because I've never used custom fields for the Relationship entity.

**Comment by danaskallman** [comment]
@mecachisenros I haven't either, but seems useful in some ways. The contact is being added as an Emergency Contact relationship type and also asking who this is? Like aunt, sister, brother, etc. Seems to make sense to add custom field there and not in contact record itself. Thoughts?
